<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FontTuner</class>
 <widget class="QWidget" name="FontTuner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFontComboBox" name="fontBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="letterSpacing">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>line spacing</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="lineSpacing">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>letter spacing</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="boldBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>bold</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="italicBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>italic</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="underlineBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>underscore</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="wordSpacing">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>word spacing</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="fontSize">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>font size</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
